### ISSUE TITLE
Release Condition conditionType is an object but should be an enum

### ISSUE THREAD

**Issue by nbihan-mediware** [hkarthik7/azure-devops-java-sdk#39]
With version 5.0.2 the API parsing fails for an expanded release with tasks. This is due to Jackson failing to deserialize the ConditionType in the release condition.
This is call
```java
Release expandedRelease = azDClientApi.getReleaseApi().getRelease(release.getId(), SingleReleaseExpands.TASKS);
```
 is throwing
```
ApiResponseParsingException: Cannot construct instance of `org.azd.release.types.ConditionType` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('event')
 at [Source: (StringReader); line: 1, column: 3340] (through reference chain: org.azd.release.types.Release["environments"]->java.util.ArrayList[0]->org.azd.release.types.ReleaseEnvironment["conditions"]->java.util.ArrayList[0]->org.azd.release.types.ReleaseCondition["conditionType"])
```

The error comes down to the class [org.azd.release.types.Condition](https://github.com/hkarthik7/azure-devops-java-sdk/blob/37260417e52338216cf259ad78ffac033b1ccff5/azd/src/main/java/org/azd/release/types/Condition.java) referencing [org.azd.release.types.ConditionType](https://github.com/hkarthik7/azure-devops-java-sdk/blob/37260417e52338216cf259ad78ffac033b1ccff5/azd/src/main/java/org/azd/release/types/ConditionType.java) but the expected result for API version 7.1 is an enum
[ConditionType enum](https://docs.microsoft.com/en-us/javascript/api/azure-devops-extension-api/conditiontype)

See conditionType here in json response
```json
            "conditions": [
                {
                    "result": false,
                    "name": "ReleaseStarted",
                    "conditionType": "event",
                    "value": ""
                },
```

**Comment by nbihan-mediware** [comment]
@hkarthik7 Hey, what does it take to have a release on Maven Central?

**Comment by hkarthik7** [comment]
@nbihan-mediware I will try to manage it as a minor incremental release next week.

**Comment by hkarthik7** [comment]
Hiya @nbihan-mediware, Closing this issue as this is fixed in new version release [v5.0.3](https://github.com/hkarthik7/azure-devops-java-sdk/releases/tag/v5.0.3) and should be available in maven central.
